Cargo.toml

```diff
@@ -22,15 +23,15 @@ frame-system-rpc-runtime-api = { path = "pallets/system/rpc/runtime-api" }
 frame-system-benchmarking = { path = "pallets/system/benchmarking" }
 
 # DA Primitives
-da-primitives = { version = "0.4", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
-kate = { version = "0.7", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
-kate-recovery = { version = "0.8", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
+da-primitives = { version = "0.4", git="https://github.com/maticnetwork/avail-core",  rev = "5bd30863f15cca7b0158319979202650dc775d04" } 
```

Comment:
We need to fix this deps.
We have to use `availproject` and specific `tag` instead of `rev`